common/log/tags.go

```diff
@@ -0,0 +1,154 @@
+	TagSysScheduleAttempt = newTagType("sys-schedule-attempt", valueTypeInteger)
+
+	// size limit
+	TagSysWorkflowSize     = newTagType("sys-workflow-size", valueTypeInteger)
+	TagSysSignalCount      = newTagType("sys-signal-count", valueTypeInteger)
+	TagSysHistorySize      = newTagType("sys-history-size", valueTypeInteger)
+	TagSysHistorySizeBytes = newTagType("sys-history-size-bytes", valueTypeInteger)
+	TagSysEventCount       = newTagType("sys-event-count", valueTypeInteger)
+
+	// ElastiSearch
+	TagSysESRequest = newTagType("sys-es-request", valueTypeString)
+	TagSysESKey     = newTagType("sys-es-mapping-key", valueTypeString)
+	TagSysESField   = newTagType("sys-es-field", valueTypeString)
+)
+
+// XDC related tags: prefix: xdc-
+var (
+	TagXDCClusterName       = newTagType("xdc-cluster-name", valueTypeString)
+	TagXDCSourceCluster     = newTagType("xdc-source-cluster", valueTypeString)
+	TagXDCPrevActiveCluster = newTagType("xdc-prev-active-cluster", valueTypeString)
+
+	// Kafka related
+	TagXDCTopicName    = newTagType("xdc-topic-name", valueTypeString)
+	TagXDCConsumerName = newTagType("xdc-consumer-name", valueTypeString)
+	TagXDCPartition    = newTagType("xdc-partition", valueTypeInteger)
+	TagXDCPartitionKey = newTagType("xdc-partition-key", valueTypeObject)
+	TagXDCOffset       = newTagType("xdc-offset", valueTypeInteger)
+
+	TagXDCFailoverMsg     = newTagType("xdc-failover-msg", valueTypeString)
+	TagXDCVersion         = newTagType("xdc-version", valueTypeInteger)
+	TagXDCCurrentVersion  = newTagType("xdc-current-version", valueTypeInteger)
+	TagXDCIncomingVersion = newTagType("xdc-incoming-version", valueTypeInteger)
+
+	TagXDCReplicationInfo  = newTagType("xdc-replication-info", valueTypeObject)
+	TagXDCReplicationState = newTagType("xdc-replication-state", valueTypeObject)
+)
+
+// Archival subsystem: archival-
+var (
+	// archival request tags
+	TagArchivalRequestDomainID             = newTagType("archival-request-domain-id", valueTypeString)
+	TagArchivalRequestWorkflowID           = newTagType("archival-request-workflow-id", valueTypeString)
+	TagArchivalRequestRunID                = newTagType("archival-request-run-id", valueTypeString)
+	TagArchivalRequestEventStoreVersion    = newTagType("archival-request-event-store-version", valueTypeInteger)
+	TagArchivalRequestBranchToken          = newTagType("archival-request-branch-token", valueTypeString)
+	TagArchivalRequestNextEventID          = newTagType("archival-request-next-event-id", valueTypeInteger)
+	TagArchivalRequestCloseFailoverVersion = newTagType("archival-request-close-failover-version", valueTypeInteger)
+
+	// archival tags (blob tags)
+	TagArchivalBucket  = newTagType("archival-bucket", valueTypeString)
+	TagArchivalBlobKey = newTagType("archival-blob-key", valueTypeString)
+
+	// archival tags (file blobstore tags)
+	TagArchivalFileBlobstoreBlobPath     = newTagType("archival-file-blobstore-blob-path", valueTypeString)
```

Comment:
The tags relating to archival file store can be deleted. They are not used.

Reply:
I see, I will follow up with you later.